MudletPackageManager.xml

```diff
@@ -2536,56 +2553,67 @@ mpkg.localization = {
 
 				{ 
 					command     = &quot;mpkg help&quot;,
-					description = {&quot;Displays this help text.&quot;}
+					description = {&quot;Displays this help text.&quot;},
+					action      =  &quot;mpkg.functions.printHelp()&quot;
 				},
 				{
 					command     = &quot;mpkg repo add \&lt;name\&gt; \&lt;URL\&gt; \&lt;Description\&gt;&quot;,
-					description = {&quot;Adds a new repository with name, URL and description to the database of known repositories.&quot;}
+          				description = {&quot;Adds a new repository with name, URL and description to the database of known repositories.&quot;},
+          				action      =  &quot;printCmdLine('mpkg repo add')&quot;
```

Comment:
hope this isn't annoying

Reply:
no not at all. 
The problem is more that this strange behaviour with indentation happens after I save and it looks different on every editor.

I wanted to ask what were you using when creating this scripts because if I use Mudlet (and install the xml as a Module) it's getting rid of all &quot; which would look like a mess if you wanted to check what I was changing on GitHub.

Reply:
Yes that's right, we changed the xml serialiser and it doesn't do `&quot;` anymore. A bit more readable